packages/Tabs/src/components/Panel/Panel.js

```diff
@@ -5,25 +5,35 @@ import * as sc from "./Panel.styles";
 
 const propTypes = {
   children: PropTypes.node,
-  /** Controls if the option is selected or not */
 
+  /** Controls if the option is selected or not */
   isSelected: PropTypes.bool,
+
+  /** Should unmount or not when isSelected is false */
+  shouldUnmount: PropTypes.bool,
```

Comment:
what about `preventUnmount` so it's aligned with preventDefault `javascript land`

Reply:
preventDefault is a function though, I thought we prefer to use `should` `has` for boolean value?